<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StoreItemDialog</class>
 <widget class="QDialog" name="StoreItemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lblTradeName">
       <property name="text">
        <string>Торговое наименование</string>
       </property>
       <property name="buddy">
        <cstring>edtTradeName</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="edtTradeName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblINN">
       <property name="text">
        <string>МНН</string>
       </property>
       <property name="buddy">
        <cstring>edtINN</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="edtINN"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="CItemListView" name="tblShippingInfo"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CItemListView</class>
   <extends>QTableView</extends>
   <header>library/ItemListView.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtTradeName</tabstop>
  <tabstop>edtINN</tabstop>
  <tabstop>tblShippingInfo</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>StoreItemDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>475</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>StoreItemDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>475</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
